### ISSUE TITLE
No label asssociated with the datalist to inform screen reader users that results appear below as they type  

### ISSUE THREAD

**Issue by SBleeks** [wet-boew/wet-boew#7988]
Can an ARIA label be added to inform screen reader users that "results appear below as you type"? Otherwise screen reader users will not know that a list is populating below the edit field.

**Comment by nschonni** [comment]
@SBleeks what browsers are you seeing this in? AFIK, none of the browsers that WET supports now will actually load the WET polyfill. The WET polyfil does use an `aria-live=polite` to read out the options though https://github.com/wet-boew/wet-boew/blob/master/src/polyfills/datalist/datalist.js#L53

**Comment by kodecount** [comment]
Also if this is related to data-list it super complex issue. I had a discussion about it over here https://github.com/wet-boew/wet-boew/issues/7829. Under the time frame I had I was unable to find a good solution so we avoided the data-list all together.

You may also run it browsers that have builtin support and WET enhancements do not kick in. I believe chrome is one of these.

**Comment by kodecount** [comment]
This is similar to issue: https://github.com/wet-boew/wet-boew/issues/7987
